manager/integration/tests/conftest.py

```diff
@@ -50,6 +51,12 @@ def pytest_addoption(parser):
                      default=False,
                      help="include cluster autoscaler tests (default: False)")
 
+    parser.addoption(LH_INSTALL_METHOD, action="store",
```

Comment:
So, this will be applicable for all the pipeline that we can choose if want to use manifest or helm to upgrade the Longhorn?
Generally, we use the same way of upgrading Longhorn the way we installed it, isn't it?

Reply:
We will have separate pipelines for each install method (like manifest, helm, rancher, flux, argocd).

And for each pipeline, the install method is fixed in code (unable to and no need to set this value as a Jenkins option), so the pipeline knows how to install Longhorn. 

But for upgrading Longhorn, it's done by `pytest` (`test container`) run by the `test build container`. The `test build container` knows the install method (which written in code), but `pytest` (`test container`) didn't. So we need `LH_INSTALL_METHOD` to pass the install method value into `pytest` (`test container`).

Reply:
Okay, so we should set default `LH_INSTALL_METHOD` as per the pipelines. e.g. for helm pipeline default `LH_INSTALL_METHOD` will be helm

Reply:
Yes, just like what I mentioned, it's already hard coded in each pipeline.